### PR TITLE
DOCS-442: Fixed a misformatted command (OS)

### PR DESCRIPTION
**Pull request by ctauchen**
This PR fixes a formatting error for a troubleshooting command in the section [Run commands across multiple nodes](https://projectcalico.docs.tigera.io/maintenance/troubleshoot/commands#run-commands-across-multiple-nodes)

Applies only to `master`

### CODE REVIEW

**Comment by ctauchen** [comment]
@tf-tigera Can you take a look?

**Review by bmckercher123 — APPROVED** [review]
Approved

File: usage_guide.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+
+TCP
```
Comment (caseydavenport) [inline usage_guide.md:3]: Hm, what is this doc? Was it intended to be included in this PR?

Comment (ctauchen) [reply usage_guide.md]: Nope! Not sure how that got in there ... Thanks for pointing that out.

**Comment by caseydavenport** [comment]
Not sure why the CLA bot is complaining but you're with us 😁

**Pull request merged**